### DISCUSSION TITLE
Serial port in MVVMCross (.NET Standard Library)

### DISCUSSION THREAD

**Discussion by btylmz** [MvvmCross/MvvmCross#4418] [Q&A] [answered]
I'm developing a WPF application for implementing a serial interface. I am implementing MVVM structure using **MVVMCross** framework. 

Serial port class does not exist in the .NET Standard 2.0 but there is a System.IO.Ports nuget package to use serial ports.
 
My problem begins here, i am trying to use this nuget package but whenever i use serialport class, app throws an System.IO.FileNotFoundException. In the output it says:

> Exception thrown: 'System.IO.FileNotFoundException' in MvxBeginner.Core.dll
> 
> An exception of type 'System.IO.FileNotFoundException' occurred in MvxBeginner.Core.dll but was not handled in user code

(MvxBeginner.Core is the project name)

I am not sure if this exception is related with .net standard or with MVVMCross. If anyone have any ideas i would really appreciate it.

**Comment by Cheesebaron** [comment]
Can you please provide the full stack trace?

**Comment by btylmz** [comment, marked as answer]
Well, i tried to solve this problem for a day. Problem is solved when i added the nuget package to WPF project as well. Before that i installed nuget package only to the core project.

Thank you for the response though.
